### DISCUSSION TITLE
Nginx with docker-ci-tool-stack

### DISCUSSION THREAD

**Discussion by psstefanov** [nginx-proxy/nginx-proxy#1951] [Q&A]
Hi,

I'm testing [docker-ci-tool-stack](https://github.com/marcelbirkner/docker-ci-tool-stack) and I would like to add Nginx in front as reverse proxy with SSL.
I'm very junior at working with Nginx and I hope that you could advise me how I can accomplish that scenario.

Thank you in advance.

**Comment by ethiclab** [comment]
Hello, you can see an example at our buildmachine.

https://github.com/ethiclab/buildmachine
